python/ray/tests/test_joblib.py

```diff
@@ -0,0 +1,163 @@
+import numpy as np
```

Comment:
Please add this test to the BUILD file, otherwise it will not be run in travis. You'll need to follow the format of the other ray tests in this directory.

Reply:
Done.